OpenRA.Game/Platform.cs

```diff
@@ -102,14 +102,14 @@ public static string ResolvePath(string path)
 			path = path.TrimEnd(new char[] { ' ', '\t' });
 
 			// paths starting with ^ are relative to the support dir
-			if (path.StartsWith("^"))
+			if (path.StartsWith("^", StringComparison.InvariantCulture))
```

Comment:
I went with this only because it's what we use everywhere else.  I didn't put any thought into the different options or whether the other places were doing it correctly.